ops/terraform/modules/mgmt_stateful/main.tf

```diff
@@ -77,27 +75,19 @@ data "aws_security_group" "management" {
 #
 # Build a VPC private local zone for CNAME records
 #
+
 resource "aws_route53_zone" "local_zone" {
   name    = "bfd-${var.env_config.env}.local"
   vpc {
     vpc_id = data.aws_vpc.main.id
   }
 }
 
-# S3 Admin bucket for logs and other adminstrative 
-#
-module "admin" {
-  source              = "../resources/s3"
-  role                = "admin"
-  env_config          = local.env_config
-  kms_key_id          = data.aws_kms_key.master_key.arn
-  acl                 = "log-delivery-write"
-}
-
 # IAM policy to allow read access to ansible vault password
 #
+
 resource "aws_iam_policy" "ansible_vault_pw_ro_s3" {
-  name        = "bfd-ansible-vault-pw-ro-s3"
+  name        = "bfd-${var.env_config.env}-ansible-vault-pw-ro-s3"
```

Comment:
Is renaming this going to affect any other modules that use this policy?  Worth a sanity check.